control.sh

```diff
@@ -1,39 +1,41 @@
 #!/bin/bash
 
-# SoftiCAR GitHub Runner - Systemd Service Management Script
+# SoftiCAR GitHub Runner - Systemd Service Control Script
 #
 # Facilitates installation, administration and uninstallation of the
 # systemd service that manages the SoftiCAR GitHub Runner lifecycle.
 #
 # Run without parameters for a list of available commands.
+#
+# TODO migrate "install" and "uninstall" to setup.sh - those have no place in here
 
 SELF_PATH=$(cd `dirname $0` && pwd)
 SERVICE_FILE=softicar-github-runner.service
 SERVICE_FILE_DESTINATION=/etc/systemd/system/$SERVICE_FILE
-SERVICE_TEMPLATE=softicar-github-runner.service-template
-SERVICE_SCRIPT_PATH=$SELF_PATH/softicar-github-runner.sh
-SERVICE_SCRIPT_ENVIRONMENT_FILE=softicar-github-runner.env
+SERVICE_TEMPLATE_PATH=systemd-service/softicar-github-runner.service-template
+SERVICE_SCRIPT_PATH=$SELF_PATH/systemd-service/softicar-github-runner-service.sh
+SERVICE_SCRIPT_ENVIRONMENT_FILE=softicar-github-runner-service.env
 RUNNER_NAME_DEFAULT="softicar-github-runner"
 RUNNER_NAME_REGEX="[a-zA-Z0-9]([_.-]?[a-zA-Z0-9]+)*"
 RUNNER_LABELS_DEFAULT="ephemeral,dind"
-REPOSITORY_NAME_EXAMPLE="SoftiCAR/some-repo"
+REPOSITORY_NAME_EXAMPLE="softicar/some-repo"
```

Comment:
somehow, this is half-baked, suggestion: either as template `REPOSITORY_NAME_EXAMPLE="<owner>/<repository>` or full example `REPOSITORY_NAME_EXAMPLE="softicar/platform`

Reply:
going with the former